dbt_project.yml

```diff
@@ -1,6 +1,6 @@
 config-version: 2
 name: 'stripe_source'
-version: '0.9.3'
+version: '0.10.0'
 require-dbt-version: [">=1.3.0", "<2.0.0"]
```

Comment:
Can we also include the `dispute` and `transfer` sources in the variables below?

Lines 14 - 32

Reply:
added